test/extensions/homotopy_continuation.jl

```diff
@@ -2,14 +2,17 @@
 using Catalyst, Test
 import HomotopyContinuation
 
+# Fetch test functions.
+include("../test_functions.jl")
+
 ### Run Tests ###
 
 # Tests for network without conservation laws.
 # Tests for Symbol parameter input.
 # Tests for Symbolics initial condition input.
 # Tests for different types (Symbol/Symbolics) for parameters and initial conditions.
 # Tests that attempts to find steady states of system with conservation laws, while u0 is not provided, gives an error.
-let 
+@test_broken let # Requires https://github.com/SciML/ModelingToolkit.jl/issues/2566 to be fixed
```

Comment:
One of the tests that are currently broken. Basically, we cannot do homotopy continuation on systems with conservation laws until that issue gets sorted out.